server/inspect.go

```diff
@@ -160,14 +160,14 @@ func (s *Server) GetExtendInterfaceMux(enableProfile bool) *chi.Mux {
 		containerID := chi.URLParam(req, "id")
 		ci, err := s.getContainerInfo(ctx, containerID, s.GetContainer, s.getInfraContainer, s.getSandbox)
 		if err != nil {
-			switch err {
-			case errCtrNotFound:
+			//nolint:gocritic // intentional if else chain
```

Comment:
What about switch, still, but without expression?

```go
switch {
case errors.Is(err, errCtrNotFound):
	http.Error(w, "can't find the container with id "+containerID, http.StatusNotFound)
case rrors.Is(err, errCtrStateNil):
	http.Error(w, "can't find container state for container with id "+containerID, http.StatusInternalServerError)
case errors.Is(err, errSandboxNotFound):
	http.Error(w, "can't find the sandbox for container id "+containerID, http.StatusNotFound)
default:
	http.Error(w, err.Error(), http.StatusInternalServerError)
}
```

Reply:
That works, fixed as suggested :+1: